Horan_Colby/Assignments/project_tdd/project_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'project'
+
+RSpec.describe Project do
+  @name = "Project Name"
+  @description = "I am a project"
+	it 'has a getter and setter for name attribute' do
+		project = Project.new("#{@name}", "#{@description}")
```

Comment:
you're repeating the same project variable a couple times.  When you need the same variable in multiple tests you should just add a before do prior to the tests.  i.e.
before do
     @project = Project.new("name", "description")
end
now in your tests you can just reference @project